components/01-atoms/text/text.stories.js

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 
-import headings from './headings/headings.twig';
+import heading from './headings/_heading.twig';
 import blockquote from './text/02-blockquote.twig';
 import pre from './text/05-pre.twig';
 import paragraph from './text/03-inline-elements.twig';
 
 import blockquoteData from './text/blockquote.yml';
+import headingData from './headings/headings.yml';
 
 /**
  * Storybook Definition.
  */
 export default { title: 'Atoms/Text' };
 
+const headings = headingData.map(d => heading(d));
```

Comment:
Can we make this line as follows to remove the commas:

`const headings = headingData.map(d => heading(d)).join('');`

Reply:
I'd also like to see a comment here about what this is doing.

Since this is an example project, and many of our Drupal users won't necessarily have used a map before, something simple like this would be helpful:

```
// Loop over each item in headingData so that we can show them all in the example(s) below
const headings = headingData.map(d => heading(d)).join('');
```

Reply:
I've made this update.